Source/WebCore/Modules/webxr/WebXROpaqueFramebufferCocoa.cpp

```diff
@@ -114,12 +114,17 @@ WebXROpaqueFramebuffer::~WebXROpaqueFramebuffer()
 {
     releaseAllDisplayAttachments();
 
-    if (RefPtr gl = m_context.graphicsContextGL()) {
+    RefPtr refContext = m_context.get();
```

Comment:
This is basically the same as `protectedContext` variables like we used to have and that we now discourage.